libopenage/pathfinding/cost_field.h

```diff
@@ -64,25 +65,32 @@ class CostField {
 	 *
 	 * @param pos Coordinates of the cell (relative to field origin).
 	 * @param cost Cost to set.
+	 * @param changed Time at which the cost value is changed.
```

Comment:
Remember to change the argument name in the docstring too! ;)

```suggestion
	 * @param valid_until Time at which the cost value is changed.
```